### PR TITLE
Add funtion that picks image from photo gallery to ui-utils

### PR DESCRIPTION
**Pull request by beneditomauro**
| - | - |
| --- | --- |
| Issue | https://github.com/ripe-tech/ripe-robin-revamp/issues/241 |

### CODE REVIEW

File: react/util/ui-utils.js [hunk 1]
```diff
@@ -159,3 +131,47 @@ export const isTablet = function () {
 export const isMobile = function () {
     return !isTablet();
 };
+
+const _handlePickImage = (resolve, reject, response, camera = true) => {
```
Comment (beneditomauro) [inline react/util/ui-utils.js:135]: { message = "..." } = {}

**Comment by beneditomauro** [comment]
Tiny extra in this PR:

- [Commit](https://github.com/ripe-tech/ripe-components-react-native/pull/168/commits/c974a094e936e30dc0c537bef45230a59af17060) that adds missing icon to be used in [ripe-robin-revamp part of the implementation ](https://github.com/ripe-tech/ripe-robin-revamp/pull/242)of this issue

File: react/assets/icons/photos.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" viewBox="0 0 24 24" aria-labelledby="imageIconTitle" stroke="#2329D6" stroke-width="1" stroke-linecap="square" stroke-linejoin="miter" fill="none" color="#2329D6"> <title id="imageIconTitle">Image</title> <rect width="18" height="18" x="3" y="3"/> <path stroke-linecap="round" d="M3 14l4-4 11 11"/> <circle cx="13.5" cy="7.5" r="2.5"/> <path stroke-linecap="round" d="M13.5 16.5L21 9"/> </svg>
```
Comment (gcandal) [inline react/assets/icons/photos.svg:1]: Where did this SVG come from and where is it being used?

Comment (beneditomauro) [reply react/assets/icons/photos.svg:1]: Coming from Ikonate

Used here in this PR: 


Tried to explain with [my comment](https://github.com/ripe-tech/ripe-components-react-native/pull/168#issuecomment-826899863) above

Comment (gcandal) [reply react/assets/icons/photos.svg:1]: I thought we already had everything from Ikonate, weird

**Review by gcandal — APPROVED** [review]

**Comment by gcandal** [comment]
@beneditomauro add this to the https://github.com/ripe-tech/ripe-components-react-native/blob/master/CHANGELOG.md

**Review by joamag — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Pull request merged**